src/database_types_file.ts

```diff
@@ -58,14 +62,44 @@ export interface Database {
           extra_information?: string
           family_id?: string
           feminine_products?: string[]
+          flag_attention?: boolean
           full_name?: string
           other_items?: string[]
           pet_food?: string[]
           phone_number?: string
           primary_key?: string
+          signposting_call?: boolean
         }
         Relationships: []
       }
+      events: {
+        Row: {
+          event_name: string
```

Comment:
I presume `event_name` is the description of a parcel event - for example, "Shopping List Printed"?